src/cryptonote_core/cryptonote_tx_utils.cpp

```diff
@@ -150,12 +150,17 @@ namespace cryptonote
       r = crypto::derive_public_key(derivation, no, miner_address.m_spend_public_key, out_eph_public_key);
       CHECK_AND_ASSERT_MES(r, false, "while creating outs: failed to derive_public_key(" << derivation << ", " << no << ", "<< miner_address.m_spend_public_key << ")");
 
-      txout_to_key tk;
-      tk.key = out_eph_public_key;
+      uint64_t amount;
+      summary_amounts += amount = out_amounts[no];
+
+      bool use_view_tags = hard_fork_version >= HF_VERSION_VIEW_TAGS;
+      crypto::view_tag view_tag = AUTO_VAL_INIT(view_tag);
```

Comment:
Is there an advantage to `AUTO_VAL_INIT()` over uniform initialization (C++11)?
```
crypto::view_tag view_tag{};
```

Does it even need to be initialized here?

Reply:
Was following the pattern from higher up in the function. But, no I don't believe there is an advantage in this case, especially considering the purpose of `AUTO_VAL_INIT` as I understand it is to use a consistent initialization across compilers, yet view tags are initialized elsewhere in the code without it. Probably was unnecessary on my part. I don't have a preference, I don't mind switching to uniform initialization.

It needs to be initialized at the same scope as `set_tx_out` a bit below.